### PR TITLE
Create MulAddSubThreads.java

### PR DESCRIPTION
**Pull request by gokulmanohar**
First file commit for MultiThreading directory. Response for Issue (#1926).

### **Describe your change:**

- [x] Add an algorithm?
- [ ] Fix a bug or typo in an existing algorithm?
- [ ] Documentation change?

Created a new Multithreading directory for including multithreading programs. Included an arithmetic program for finding sum, product and difference of two operands using multithreading on java.

### **Checklist:**

- [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Java/blob/master/CONTRIBUTING.md).
- [x] This pull request is all my own work -- I have not plagiarized.
- [x] I know that pull requests will not be merged if they fail the automated tests.
- [x] This PR only changes one algorithm file. To ease review, please open separate PRs for separate algorithms.
- [x] All functions and variable names follow Java naming conventions.
- [x] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
- [x] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

File: MultiThreading/MulAddSubThreads.java [hunk 1]
```diff
@@ -0,0 +1,63 @@
+import java.util.Scanner;
+public class MulAddSubThreads{
```
Comment (prashantdoshi28) [inline MultiThreading/MulAddSubThreads.java:2]: Please add proper documentation for your code.

**Review by prashantdoshi28 — CHANGES_REQUESTED** [review]
Make sure all checks are passing for PR.

File: MultiThreading/MulAddSubThreads.java [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+		catch (Exception e) {
+		    e.printStackTrace();
+		}
+		finally{
+			sc.close();
+		}
+	}
+}
+ 
+class Mul extends Thread {
```
Comment (ayaankhan98) [inline MultiThreading/MulAddSubThreads.java:26]: better to use `Runnable` Interface

File: MultiThreading/MulAddSubThreads.java [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+		this.a = p;
+		this.b = q;
+	}
+	public void run() {
+        value =  a * b;
+        System.out.println ("Multiplication: " +value); 
+	}
+}
+
+class Add extends Thread {
```
Comment (ayaankhan98) [inline MultiThreading/MulAddSubThreads.java:38]: same goes here

File: MultiThreading/MulAddSubThreads.java [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+		this.a = p;
+		this.b = q;
+	}
+	public void run() {
+        value = a + b;
+        System.out.println ("Addition: " +value); 
+	}
+}
+
+class Sub extends Thread {
```
Comment (ayaankhan98) [inline MultiThreading/MulAddSubThreads.java:50]: here too.

**Pull request closed without merging**